src/map/entities/battleentity.cpp

```diff
@@ -512,7 +512,7 @@ int16 CBattleEntity::addTP(int16 tp)
 
 /************************************************************************
  *                                                                      *
- *  Изменяем количество жизней сущности                                 *
+ *  Change the number of lives of the entity                            *
```

Comment:
`Change current HP of entity` if kept at all

Reply:
Removing, thanks.